src/preferences/dialog/dlgprefcontrols.cpp

```diff
@@ -125,13 +125,27 @@ DlgPrefControls::DlgPrefControls(QWidget * parent, MixxxMainWindow * mixxx,
     connect(buttonGroupKeyLockMode, SIGNAL(buttonClicked(QAbstractButton*)),
             this, SLOT(slotKeyLockMode(QAbstractButton *)));
 
+    // 0 Lock original key, 1 Lock current key
     m_keylockMode = m_pConfig->getValue(
         ConfigKey("[Controls]", "keylockMode"), 0);
     foreach (ControlProxy* pControl, m_keylockModeControls) {
         pControl->set(m_keylockMode);
     }
 
     //
+    // Key unlock mode
+    //
+    connect(buttonGroupKeyUnlockMode, SIGNAL(buttonClicked(QAbstractButton*)),
+            this, SLOT(slotKeyUnlockMode(QAbstractButton *)));
+
+    // 0 Keep locked key, 1 Reset locked key
+    m_keyunlockMode = m_pConfig->getValue(
+        ConfigKey("[Controls]", "keyunlockMode"), 0);
```

Comment:
Am I right that default is set here:
If [Conrols],keyunlockMode doesn't return anything, it's set to "0"?
Should be "1" if I leave GUI as it is